### PR TITLE
APIS-4663 - Update to Play 2.6

### PR DESCRIPTION
**Pull request by peteslater-ee**

### CODE REVIEW

File: app/uk/gov/hmrc/testuser/MicroserviceModule.scala [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import uk.gov.hmrc.play.bootstrap.http.DefaultHttpClient
+import uk.gov.hmrc.testuser.services.PasswordConfig
+
+class MicroserviceModule(val environment: Environment, val configuration: Configuration) extends AbstractModule {
+
+  def configure(): Unit = {
+    val appName = "api-platform-test-user"
+    Logger(getClass).info(s"Starting microservice : $appName : in mode : ${environment.mode}")
+
+    bind(classOf[HttpGet]).to(classOf[DefaultHttpClient])
```
Comment (AndySpaven) [inline app/uk/gov/hmrc/testuser/MicroserviceModule.scala]: Are these needed?

File: build.sbt [hunk 2]
```diff
@@ -8,31 +8,43 @@ import uk.gov.hmrc.versioning.SbtGitVersioning
...
 lazy val appName = "api-platform-test-user"
 lazy val appDependencies: Seq[ModuleID] = compile ++ test
+lazy val akkaVersion = "2.5.23"
+lazy val akkaHttpVersion = "10.0.15"
 
 lazy val compile = Seq(
-  "uk.gov.hmrc" %% "bootstrap-play-25" % "5.1.0",
-  "uk.gov.hmrc" %% "play-ui" % "8.3.0-play-25",
-  "uk.gov.hmrc" %% "play-hmrc-api" % "3.4.0-play-25",
+  "uk.gov.hmrc" %% "bootstrap-play-26" % "1.4.0",
```
Comment (AndySpaven) [inline build.sbt:14]: I think many of the play libs are transitive deps of bootstrap

File: build.sbt [hunk 3]
```diff
@@ -8,31 +8,43 @@ import uk.gov.hmrc.versioning.SbtGitVersioning
...
 lazy val test = Seq(
-  "uk.gov.hmrc" %% "hmrctest" % "3.9.0-play-25" % scope,
-  "uk.gov.hmrc" %% "reactivemongo-test" % "4.15.0-play-25" % scope,
+  "uk.gov.hmrc" %% "hmrctest" % "3.9.0-play-26" % scope,
+  "uk.gov.hmrc" %% "reactivemongo-test" % "4.15.0-play-26" % scope,
   "org.scalatest" %% "scalatest" % "3.0.4" % scope,
-  "org.scalatestplus.play" %% "scalatestplus-play" % "2.0.0" % scope,
+  "org.scalatestplus.play" %% "scalatestplus-play" % "3.1.2" % scope,
   "org.pegdown" % "pegdown" % "1.6.0" % scope,
   "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,
```
Comment (AndySpaven) [inline build.sbt]: I think this is transitive dep of hmrc test

File: test/it/uk/gov/hmrc/testuser/PlatformIntegrationSpec.scala [hunk 4]
```diff
@@ -34,7 +34,8 @@ class PlatformIntegrationSpec extends UnitSpec with ScalaFutures with GuiceOneAp
   trait Setup {
     implicit def mat: akka.stream.Materializer = app.injector.instanceOf[akka.stream.Materializer]
 
-    val documentationController = new DocumentationController(LazyHttpErrorHandler) {}
+//    val documentationController = new DocumentationController(LazyHttpErrorHandler) {}
```
Comment (AndySpaven) [inline test/it/uk/gov/hmrc/testuser/PlatformIntegrationSpec.scala]: remove this comment

**Review by AndySpaven — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/testuser/MicroserviceModule.scala (hunk 1 above)
Comment (peteslater-ee) [reply app/uk/gov/hmrc/testuser/MicroserviceModule.scala]: Nope - gone

File: build.sbt (hunk 2 above)
Comment (peteslater-ee) [reply build.sbt:14]: play-hmrc-api appears to be
play-ui isn't

File: build.sbt (hunk 3 above)
Comment (peteslater-ee) [reply build.sbt]: Nah

File: test/it/uk/gov/hmrc/testuser/PlatformIntegrationSpec.scala (hunk 4 above)
Comment (peteslater-ee) [reply test/it/uk/gov/hmrc/testuser/PlatformIntegrationSpec.scala]: Done

**Pull request merged**